components/scream/src/share/field/field_property_checks/field_lower_bound_check.hpp

```diff
@@ -26,14 +21,12 @@ class FieldLowerBoundCheck: public FieldWithinIntervalCheck<RealType> {
   // can repair fields that fail the check by overwriting nonpositive values
   // with the given lower bound. If can_repair is false, the check cannot
   // apply repairs to the field.
-  explicit FieldLowerBoundCheck (const_RT lower_bound,
+  explicit FieldLowerBoundCheck (const double lower_bound,
```

Comment:
Any reason not to use `Real` here?

Reply:
That makes sense. No reason to use Real here unless we mind the implicit cast from Real to double in the constructor.